_i18n/en/index.md

```diff
@@ -28,7 +28,7 @@ QMK is developed and maintained by Jack Humbert of OLKB with contributions from
 
 ## Documentation
 
-[Documentation](https://docs.qmk.fm) is hosted on [Gitbook](https://www.gitbook.com/book/qmk/firmware/details) and [Github](https://github.com/qmk/qmk_firmware/tree/master/docs) (they are synced). You can request changes by making a fork and [pull request](https://github.com/qmk/qmk_firmware/pulls) or by clicking the "suggest an edit" on any page.
+[Documentation](https://docs.qmk.fm) is hosted on [Gitbook](https://docs.qmk.fm) and [Github](https://github.com/qmk/qmk_firmware/tree/master/docs) (they are synced). You can request changes by making a fork and [pull request](https://github.com/qmk/qmk_firmware/pulls) or by clicking the "suggest an edit" on any page.
```

Comment:
```suggestion
[The official documentation](https://docs.qmk.fm) is powered by [Docsify](https://docsify.js.org/) and hosted on [GitHub](https://github.com/qmk/qmk_firmware/tree/master/docs). You can request changes by making a fork and [pull request](https://github.com/qmk/qmk_firmware/pulls) or by clicking the "Edit Document" link at the bottom of any page.
```

We don't use Gitbook anymore - this suggestion brings the text more in line with what's currently in the qmk_firmware readme.